jujugui/static/gui/src/app/components/charmbrowser/test-charmbrowser.js

```diff
@@ -130,6 +130,7 @@ describe('Charmbrowser', function() {
           charmstoreURL="http://1.2.3.4/"
           gisf={true}
           setPageTitle={setPageTitle}
+          showExperts={undefined}
```

Comment:
I had this as undefined because if there are no flags then this could be undefined instead of false. But yes, it should be false....if we set up a getter for flags that might be a nice thing :)